docs/topics/native/native-objc-interop.md

```diff
@@ -136,67 +147,70 @@ package my.library
+* [Top-level read-only properties](https://github.com/kotlin-hands-on/kotlin-swift-interopedia/blob/main/docs/functionsandproperties/Top-level%20val%20properties.md)
+* [Top-level mutable properties](https://github.com/kotlin-hands-on/kotlin-swift-interopedia/blob/main/docs/functionsandproperties/Top-level%20mutable%20var%20properties.md)
+
 ### Method names translation
 
-Generally, Swift argument labels and Objective-C selector pieces are mapped to Kotlin
-parameter names. These two concepts have different semantics, so sometimes
-Swift/Objective-C methods can be imported with a clashing Kotlin signature. In this case,
-the clashing methods can be called from Kotlin using named arguments, e.g.:
+Generally, Swift argument labels and Objective-C selector pieces are mapped to Kotlin parameter names. These two concepts
+have different semantics, so sometimes Swift/Objective-C methods can be imported with a clashing Kotlin signature.
+In this case, the clashing methods can be called from Kotlin using named arguments, for example:
 
 ```swift
 [player moveTo:LEFT byMeters:17]
 [player moveTo:UP byInches:42]
 ```
 
-In Kotlin, it would be:
+In Kotlin, it's:
 
 ```kotlin
 player.moveTo(LEFT, byMeters = 17)
 player.moveTo(UP, byInches = 42)
 ```
 
-The methods of `kotlin.Any` (`equals()`, `hashCode()` and `toString()`) are mapped 
-to the methods `isEquals:`, `hash` and `description` in Objective-C, and to the method
-`isEquals(_:)` and the properties `hash`, `description` in Swift.
+Here's how  some methods of `kotlin.Any` are mapped to Swift/Objective-C:
 
-You can specify a more idiomatic name in Swift or Objective-C, instead of renaming the Kotlin declaration.
-Use the `@ObjCName` annotation that instructs the Kotlin compiler to use a custom Objective-C and Swift name for methods
-or parameters.
+| Kotlin       | Swift          | Objective-C   |
+|--------------|----------------|---------------|
+| `equals()`   | `isEquals(_:)` | `isEquals:`   |
+| `hashCode()` | `hash`         | `hash`        |
+| `toString()` | `description`  | `description` |
 
-> Using this annotation requires [opt-in](opt-in-requirements.md).
->
-{type="note"}
+[See an example with data classes in the Kotlin-Swift interopedia](https://github.com/kotlin-hands-on/kotlin-swift-interopedia/blob/main/docs/classesandinterfaces/Data%20classes.md).
+
+You can specify a more idiomatic name in Swift or Objective-C, instead of renaming the Kotlin declaration with
+the [`@ObjCName` annotation](#custom-declaration-names).
 
 ### Errors and exceptions
 
-Kotlin has no concept of checked exceptions, all Kotlin exceptions are unchecked.
-Swift has only checked errors. So if Swift or Objective-C code calls a Kotlin method
-which throws an exception to be handled, then the Kotlin method should be marked
-with a `@Throws` annotation specifying a list of "expected" exception classes.
+Kotlin has no concept of checked exceptions, all Kotlin exceptions are unchecked. Swift has only checked errors.
```

Comment:
This feels like a contradiction: if there is no concept of checked exceptions, they cannot be unchecked because unchecked is the opposite of checked. If that makes sense. In any case, I think this can be shortened to say, “All Kotlin exceptions are unchecked.”

Also, it follows that for people with a Kotlin-heavy background, it might be nice to include a short aside or a link about what checked exceptions are. Maybe a link to our [concepts](https://kotlinlang.org/docs/exceptions.html#checked-exceptions), but that page, too, mentions only that checked exceptions aren't there.